src/containers/App/component.js

```diff
@@ -64,12 +46,13 @@ export default class App extends React.Component {
   }
 
   getChildContext () {
-    const { dispatch, currentUser, isMobile, location } = this.props
-    return {dispatch, currentUser, isMobile, location}
+    const { currentUser, isMobile, location } = this.props
+    const { dispatch } = this.context
```

Comment:
I get that completely now, but for some reason I thought that Redux put dispatch into context for anything connected. My bad. Will fix.